rules/eslint/style/walmart.js

```diff
@@ -78,6 +80,8 @@ module.exports = {
     "quote-props": 0,
     // specify whether double or single quotes should be used
     "quotes": [2, "double"],
+    // require identifiers to match the provided regular expression
+    "id-match": 0,
```

Comment:
This seems like a great one to add, it would throw lint errors for variable names like `partiallyCamel_cased` or `function annoying_dev_made_me () {}`.